### PR TITLE
Batch validator status requests

### PR DESCRIPTION
**Pull request by cemozerr**
## PR Description
Batch validator status requests.

## Documentation

- [x] I thought about documentation and added the `documentation` label to this PR if updates are required.

### CODE REVIEW

File: validator/remote/src/main/java/tech/pegasys/teku/validator/remote/RemoteValidatorApiHandler.java [hunk 1]
```diff
@@ -99,54 +100,42 @@ public RemoteValidatorApiHandler(
...
+          publicKeys.subList(i, Math.min(publicKeys.size(), i + MAX_PUBLIC_KEY_BATCH_SIZE));
+      requestValidatorObject(batch, validatorResponseTFunction).ifPresent(returnedObjects::putAll);
+    }
+    return returnedObjects;
   }
 
-  private Optional<Map<BLSPublicKey, Integer>> requestValidatorIndices(
-      final List<BLSPublicKey> batch) {
+  private <T> Optional<Map<BLSPublicKey, T>> requestValidatorObject(
+      final List<BLSPublicKey> batch, Function<ValidatorResponse, T> validatorResponseTFunction) {
```
Comment (ajsutton) [inline validator/remote/src/main/java/tech/pegasys/teku/validator/remote/RemoteValidatorApiHandler.java]: nit: `validatorResponseTFunction` isn't a great name.  Maybe `valueExtractor`?

**Review by ajsutton — APPROVED** [review]
LGTM.

File: validator/remote/src/main/java/tech/pegasys/teku/validator/remote/RemoteValidatorApiHandler.java (hunk 1 above)
Comment (cemozerr) [reply validator/remote/src/main/java/tech/pegasys/teku/validator/remote/RemoteValidatorApiHandler.java]: blame intellij!

Comment (cemozerr) [reply validator/remote/src/main/java/tech/pegasys/teku/validator/remote/RemoteValidatorApiHandler.java]: done.

**Pull request merged**